playground/src/components/echo-server/echo-server.js

```diff
@@ -30,12 +31,24 @@ const EchoServer = () => {
   const echoServerApiUrl = useApplicationContext(
     (context) => context.environment.echoServerApiUrl
   );
+  const [
+    shouldIncludeParamInRequest,
+    setShouldIncludeParamInRequest,
+  ] = useState(false);
+
+  const onChangeShouldIncludeParamInRequest = (event) => {
+    const nextValue = event.target.value === 'false' ? true : false;
+    setShouldIncludeParamInRequest(nextValue);
+  };
+
   const handleSendRequest = useCallback(() => {
+    const searchParam = shouldIncludeParamInRequest ? `?query=value` : null;
```

Comment:
The `?` is already assigned when joining the values below.

```suggestion
    const searchParam = shouldIncludeParamInRequest ? `hello=world` : null;
```